core/src/main/scala/scalaz/syntax/ApplicativeErrorSyntax.scala

```diff
@@ -0,0 +1,30 @@
+package scalaz
+package syntax
+
+/** Wraps a value `self` and provides methods related to `ApplicativeError` */
+final class ApplicativeErrorOps[F[_], S, A] private[syntax](self: F[A])(implicit val F: ApplicativeError[F, S]) {
+  ////
+
```

Comment:
https://github.com/scalaz/scalaz/blob/e8ae7157a89a9fdf020fa65f7e6d0c396d77075e/core/src/main/scala/scalaz/syntax/MonadErrorSyntax.scala

We should move some methods from `MonadError`.